File: Golang/453. Minimum Moves to Equal Array Elements.go
func minMoves(nums []int) int { // 48ms
    length := len(nums)
    min := nums[0]
    count := 0
    for _, num := range nums {
        count += num
        if min > num {
            min = num
        }
    }
    return count - min*length
}

File: Python3/453. Minimum Moves to Equal Array Elements.py
class Solution:
    def minMoves(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        # return sum(nums) - min(nums)*len(nums) # 96ms
        
        length = len(nums) # 68ms
        count = 0
        min_num = nums[0]
        for num in nums:
            count += num
            if num < min_num:
                min_num = num
        return count - min_num * length
